asm/buffer_test.go

```
@@ -7,6 +7,13 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestNewBuffer(t *testing.T) {
+	buffer, err := NewBuffer(1)
+	require.NoError(t, err)
+	defer buffer.Free()
```

golangci-lint (2.12.2)

[error] Error return value of `buffer.Free` is not checked

(errcheck)